public/app/themes/clarity/src/components/c-intranet-switcher/view.php

```diff
@@ -7,6 +7,13 @@
 $current_intranet = get_intranet_code();
 
 $referrer = (isset($_SERVER['HTTP_REFERER']) ? parse_url($_SERVER['HTTP_REFERER']) : parse_url(get_home_url()));
```

Comment:
Hey @EmilyHazlehurst I know you didn't write this code, but do you know why we are using `HTTP_REFERER`?

It looks like there's some logic and it's being appended to the href of internal URLs - I'm not really sure why.

Reply:
@EarthlingDavey No, I'm not sure why either. It looks like maybe once there were additional parameters being added?

I've removed it as it doesn't seem to do anything.